classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
```

Comment:
- [ ] there needs to be an attribute reader for student as well in this class.

Reply:
done